proto/evidence_store.proto

```diff
@@ -29,20 +29,33 @@ syntax = "proto3";
 package clouditor;
 
 import "evidence.proto";
+import "google/api/annotations.proto";
 import "google/protobuf/empty.proto";
 
 option go_package = "api/evidence";
 
 // Manages the storage of evidences
 service EvidenceStore {
-  // Stores an evidence to the evidence storage
-  rpc StoreEvidence(Evidence) returns (StoreEvidenceResponse);
+  // Stores an evidence to the evidence storage. Part of the public API, also exposed at REST.
+  rpc StoreEvidence(Evidence) returns (StoreEvidenceResponse) {
```

Comment:
Shoudn't this now be a `StoreEvidenceRequest`?

Reply:
Yes, wait for the new commit :)